### PR TITLE
Update custom.css

### PR DESCRIPTION
**Pull request by mithlesh4257**
Fix issue #158
Responsiveness issue:design and alignment of input boxes is not good for all devices.

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -51,10 +51,11 @@
   margin-bottom: 50px !important;
 }
 .newsletter-name{
-  padding-right: 0px;
+  padding-right: 5px !important;
```
Comment (0x48piraj) [inline css/custom.css:54]: Why used `!important`
It's not healthy.

Comment (mithlesh4257) [reply css/custom.css:54]: Okay @0x48piraj , I should try to solve it otherway around.

Comment (mithlesh4257) [reply css/custom.css:54]: @0x48piraj No other option.
That's one only.

See I have uploaded the different images of different screen size

f1
f2

It's working.

Comment (mithlesh4257) [reply css/custom.css:54]: @0x48piraj kindly review it.

**Comment by mariobehling** [comment]
As all buttons on the website have a round shape at the corner, this shape should also stay round.

**Comment by mariobehling** [comment]
Issue is partly closed already. Please follow up in new PR/issue.

**Pull request closed without merging**